### DISCUSSION TITLE
Decompiler text colors for globals/variables

### DISCUSSION THREAD

**Discussion by as0ni** [NationalSecurityAgency/ghidra#5128] [Q&A]
Two questions about the attached screenshot:
1. Should all "DAT" labels be green since they are all globals? I'm not sure why only the first is green. My understanding is that yellow is for local variables (which they are not).
2. What does the "_" (underscore) at the beginning of a label represent?

Thanks!

![RE_VM](https://user-images.githubusercontent.com/8812157/63172671-3bfba980-c00c-11e9-9496-cdc9d9bbb6ca.jpg)

**Comment by cod1raph** [comment]
ftr i guess _DAT yellow = ghidra finds writes to it but no reads
